tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -76,7 +76,7 @@ def delete_request(self, url):
         # We don't return anything for a delete
         self._make_request(self.parent_srv.session.delete, url, auth_token=self.parent_srv.auth_token)
 
-    def put_request(self, url, xml_request, content_type='text/xml'):
+    def put_request(self, url, xml_request=None, content_type='text/xml'):
```

Comment:
So cancel takes an empty (as in `b''`) PUT? Didn't run job take an empty `<tsRequest/ >`?

Reply:
yeah...